src/site/styles/index.md-njk

```diff
@@ -18,12 +18,11 @@ the style of buttons or adjust the thickness of borders on form inputs.
 
 ## Typography, spacing and layout
 
-The [MOD.UK Figma Design Library]({{ vars.figmaURLs.designLibrary }}) shows how
-to apply:
+Some of our guides are in the [MOD.UK Figma Design Library]({{ vars.figmaURLs.designLibrary }}) – the following links open in new windows in Figma:
 
-- [typography]({{vars.figmaURLs.typography}})
-- [spacing]({{vars.figmaURLs.spacing}})
-- [layout]({{vars.figmaURLs.layout}})
+- <a rel="noreferrer noopener" target="_blank" href="{{ vars.figmaURLs.typography }}">typography</a>
```

Comment:
I think we should also add `<span class="govuk-visually-hidden"> (opens in new tab)</span>` to the text of each link as suggested on https://design-system.service.gov.uk/styles/typography/#opening-links-in-a-new-tab in the bit about lots of links (I added a space as well before the opening bracket).